### ISSUE TITLE
Incorrect JSON Schema draft URI

### ISSUE THREAD

**Issue by iann0036** [aws-cloudformation/aws-cloudformation-resource-schema#83]
According to the [spec](https://github.com/json-schema-org/json-schema-spec/blob/draft-07/schema.json), the draft-07 schema URI should begin with `http://` and not `https://` (schema ID does not necessarily mean the network location where the resource is stored).

So I believe the [references](https://github.com/aws-cloudformation/aws-cloudformation-resource-schema/blob/master/src/main/resources/schema/provider.definition.schema.v1.json#L2) should be changed to reflect this.

**Comment by PatMyron** [comment]
looks like they were flipped here: https://github.com/aws-cloudformation/aws-cloudformation-resource-schema/pull/57

**Comment by jotompki** [comment]
If if remember correctly, there was some justification for the http->https  changes in the references, as the library we used did not handle http 301 redirects returned by json-schema.org. 

This issue https://github.com/json-schema-org/json-schema-org.github.io/issues/236 seems related and was resolved around the same time as #57 . Anyways, I changed them back and it looks to be working correctly with all tests passing. Will send out a PR.
